app/main/views/services.py

```diff
@@ -233,11 +241,16 @@ def import_service(service_id):
         abort(400, "Database Error: {0}".format(e))
 
     if not framework.expired:
+        try:
             search_api_client.index(
                 service_id,
                 service.data,
                 supplier.name,
                 framework.name)
+        except apiclient.HTTPError as e:
+            current_app.logger.warning(
```

Comment:
Index failures should definitely be out of the ordinary, but since we're not returning an error status code for the response I don't think we should log it as ERROR either. I might be wrong about this, but I'd keep errors for "request failed" events. So I'd expect the search-api to log an ERROR if there's something wrong.